### ISSUE TITLE
nightly test pipeline failing with indentation error

### ISSUE THREAD

**Issue by aramase** [Azure/secrets-store-csi-driver-provider-azure#884]
<img width="1302" alt="Screen Shot 2022-05-09 at 8 23 43 PM" src="https://user-images.githubusercontent.com/10608509/167437366-282f67f9-e326-40d9-b6de-2e72a46d04b3.png">

This used to work before, so I wonder if something changed in ADO that's causing this failure.

cc @nilekhc

**Comment by nilekhc** [comment]
Yeah.. Look like breaking change from ADO. I just added additional space and that worked.

https://github.com/Azure/secrets-store-csi-driver-provider-azure/pull/864/files#diff-756c780b9465b585ef3a4cd9439907ca68942baa1df6c73a92b9d8a7ae9f9c54R45-R53

**Comment by aramase** [comment]
@nilekhc Can you open a separate PR with that fix so we can get the nightly pipelines unblocked?

**Comment by aramase** [comment]
Looks like there might have been a change in ADO to fix this. The pipeline has been running again since Friday (05/13).

Closing this issue now.
